staff/almu-itur/app-template/about.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About</title>
+</head>
+<body>
+    
+    <header> 
+        <nav>
+            <ul>
+                <li><a href=".\home.html">Home</a></li>
+                <li><a href=".\profile.html">Profile</a></li>
+                <li><a href=".\about.html">About</a></li>
+                <li><a href=".\news.html">News</a></li>
+                <li><a href=".\landing.html">Landing</a></li>
+                <li><a href=".\login.html">Log in</a></li>
+                <li><a href=".\register.html">Register</a></li>
+                <li><a href=".\contact.html">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+        
+    <h1>ABOUT</h1>
+            
+    <main>
+        <a href="#home">Home</a><a href="#profile">Profile</a><a href="#about">About</a><a href="#news">News</a><a href="#contact">Contact</a><a href="#login">Log in</a><a href="#register">Register</a><a href="#landing">Landing</a>
+        <header>ABOUT</header>
+        <article>
+            <p>Lorem ipsum dolor sit amet consectetur adipiscing elit a, bibendum est parturient sociosqu facilisis eu penatibus lobortis semper, condimentum purus eget ligula odio etiam dictum. Massa augue odio penatibus lacus nec litora nullam nunc, fringilla nulla venenatis porttitor phasellus aptent curabitur, facilisis vel tempor posuere potenti etiam imperdiet. Turpis eget velit volutpat tristique class cum, natoque ut lacus rutrum risus ac praesent, vel hac sociis magnis platea.</p>
+            <p>Lorem ipsum dolor sit amet consectetur adipiscing elit a, bibendum est parturient sociosqu facilisis eu penatibus lobortis semper, condimentum purus eget ligula odio etiam dictum. Massa augue odio penatibus lacus nec litora nullam nunc, fringilla nulla venenatis porttitor phasellus aptent curabitur, facilisis vel tempor posuere potenti etiam imperdiet. Turpis eget velit volutpat tristique class cum, natoque ut lacus rutrum risus ac praesent, vel hac sociis magnis platea.</p>
+            <p>Lorem ipsum dolor sit amet consectetur adipiscing elit a, bibendum est parturient sociosqu facilisis eu penatibus lobortis semper, condimentum purus eget ligula odio etiam dictum. Massa augue odio penatibus lacus nec litora nullam nunc, fringilla nulla venenatis porttitor phasellus aptent curabitur, facilisis vel tempor posuere potenti etiam imperdiet. Turpis eget velit volutpat tristique class cum, natoque ut lacus rutrum risus ac praesent, vel hac sociis magnis platea.</p>
+        </article>
```

Comment:
may you want to use a section here?

Reply:
agree... article has no sense here, please, RTFM

Reply:
Ok